### ISSUE TITLE
[Bug]: States being resized based on rotation

### ISSUE THREAD

**Issue by TheJadeKnight** [RPTools/maptool#4944]
### Describe the Bug

When Tokens rotate, the state images on them get resized, which can make the Token look a completely different size than it is.  I have attached an example.

In this example, all of the Tokens are the same size, but they look like they're different sizes because their states get bigger when they're facing vertices.  Expected behavior would be that the states remain the same size regardless of the facing of the Token.

### To Reproduce

Add a state with an image to a token.  Rotate the token so it is facing a square vertex.

### Expected Behaviour

For states to remain the same size regardless of the facing of the Token, so it doesn't look like Tokens are all different sizes when they're rotated.

### Screenshots

![image](https://github.com/user-attachments/assets/09cfcf4f-fd6f-4ad7-a308-c38d7dd22548)

### MapTool Info

1.14.3

### Desktop

Windows

**Comment by bubblobill** [comment]
Is probably being sized relative to the bounding box. 
![image](https://github.com/user-attachments/assets/add850da-14cd-4820-903d-8c843d4bfa5f)
Does the state image size oscillate each quarter turn?

**Comment by FullBleed** [comment]
I've noticed that this is only an issue with fine rotational adjustments (rotating while holding Ctrl-Shft) or on rotations to vertices (must set to allow in preferences.)

Not a issue with quarter turns.

**Comment by TheJadeKnight** [comment]
> Is probably being sized relative to the bounding box. ![image](https://private-user-images.githubusercontent.com/45483160/371832976-add850da-14cd-4820-903d-8c843d4bfa5f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjc2MDE4MjcsIm5iZiI6MTcyNzYwMTUyNywicGF0aCI6Ii80NTQ4MzE2MC8zNzE4MzI5NzYtYWRkODUwZGEtMTRjZC00ODIwLTkwM2QtOGM4NDNkNGJmYTVmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTI5VDA5MTg0N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWY2NzRmMTQxYmZiODYzZjRmN2ZlNDVmYTA3ZGY5NGIyOTBlNThlNDIwOGY1OTkxM2NiOGVmNTVkZjNhMzhjZTEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.VGXvzte8wG8fTf6GYXkL9knUax3K3b-blSEjG-y9l-s) Does the state image size oscillate each quarter turn?

Yes, I expected it's being sized relative to the bounding box.  It's an issue for rotations to vertices, but not on the faces.

**Comment by bubblobill** [comment]
Sorted in #4968

**Comment by kwvanderlinde** [comment]
Reopening due to revert of #4968.
